_targets/build.project.armv7m7-imxrt117x

```diff
@@ -40,24 +40,71 @@ KERNEL_OFFS=$((0x11000))
 # Project specific build
 #
 
-PROGS=("dummyfs" "imxrt-multi" "psh" "imxrt-flash")
-
-
-b_mkscript() {
+PROGS=()
+BOOT_DEVICE="flash0"
+NAME_USER_SCRIPT="user.plo"
+OFFS_USER_SCRIPT=$((0x10a00))
+MAGIC_USER_SCRIPT=
+
+# Pre-init script is launched before user script
+PREINIT_SCRIPT=("map itcm 0 40000 rwx"
+                "map dtcm 20000000 20028000 rw"
+                "map ocram2 202c0000 2033fe00 rwx"
+                "map xip1 30000000 30400000 rx"
+                "phfs flash0 2.0 raw"
+                "console 0.10"
+                "wait 2000")
+
+# Obligatory applications to run Phoenix-RTOS
+USER_SCRIPT=("kernel ${BOOT_DEVICE}"
+             "app ${BOOT_DEVICE} -x dummyfs xip1 ocram2"
+             "app ${BOOT_DEVICE} -x imxrt-multi ocram2 ocram2"
+             "app ${BOOT_DEVICE} -x psh xip1 ocram2")
+
+
+b_mkscript_user() {
```

Comment:
code duplication with rt1064

Reply:
Yeah, I know. 
@pawelpisarczyk asked to not create additional files if it is not necessary so I try to avoid such situation. Currently it is only one function. @pawelpisarczyk what do you think abut creating common file? Currently it is only used by `imxrt` family but it can change in the future and can be used by other platforms.